pushServices/src/gms/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
 
     <application>
         <service
-            android:name=".core.GmsPushService"
+            android:name=".core.GmsPushServiceService"
```

Comment:
Where is this GmsPushServiceService? Is this secne "Service" required?

Reply:
Refactoring renaming issue, fixed.